application/src/test/resources/intial_delay_greater_than_repair.yml

```diff
@@ -0,0 +1,22 @@
+#
+# Copyright 2020 Telefonaktiebolaget LM Ericsson
+#
```

Comment:
Delete this new file, please

Reply:
It is used for testing a faulty confiuration